src/aaudio/AAudioLoader.cpp

```diff
@@ -146,6 +146,12 @@ int AAudioLoader::open() {
     if (getSdkVersion() >= __ANDROID_API_S_V2__) {
         stream_getChannelMask = load_U_PS("AAudioStream_getChannelMask");
     }
+
+    // TODO: Add check for __ANDROID_API_U__ after Android U release
```

Comment:
You can check for version > T

Reply:
The issue is that getSdkVersion() doesn't return 34 yet. We need to wait for Android 14 beta to come out. We are still in developer preview

Reply:
Our apps work fine without the check. If we really want this check, we can't submit this PR until we update our apps to 
Android 14.

Reply:
Added a prerelease check. It mirrors [this](https://cs.android.com/android/platform/superproject/+/master:frameworks/libs/modules-utils/java/com/android/modules/utils/build/SdkLevel.java;l=66?q=isAtLeastPreReleaseCodename) check for Java